packages/extension-chakra-storefront/src/pages/product-list/partials/refinements.jsx

```diff
@@ -41,6 +41,10 @@ const Refinements = ({
     selectedFilters,
     isLoading
 }) => {
+    const {
+        pages: {ProductList: productListConfig}
+    } = useExtensionConfig()
+    const FILTER_ACCORDION_SATE = productListConfig?.filterAccordionSate
```

Comment:
We can safely assume that `productListConfig` is defined already, because the extension's default config should have defined something for it.

I believe there are other similar cases like that one. I think I'd rather the code results in an error, rather than the end result be undefined.